openlibrary/core/fulltext.py

```diff
@@ -34,10 +34,10 @@ def fulltext_search_api(params):
 
     logger.debug('URL: ' + search_select)
     try:
-        response = requests.get(search_select, headers=headers, timeout=30)
```

Comment:
This appears to have triggered an error somehow ; I think something headers related? I think one of the dictionary values is `None`

<img width="1015" height="637" alt="image" src="https://github.com/user-attachments/assets/72d2297d-be38-4207-bc99-f7c378c4087f" />